geonotebook/kernel.py

```diff
@@ -470,9 +471,10 @@ def handle_comm_open(self, comm, msg):
         # THis should be handled in a callback that is fired off
         # When set protocol etc is complete.
         if self.initializing:
+            vis_url = Config().config.get('basemap', 'url')
```

Comment:
Can we move this inside a property on the Config() class with a default? I'd like to avoid any ConfigParser specific syntax outside of config.py